exercises/static/exercises/drone_cat_mouse/web-template/gui.py

```diff
@@ -141,8 +141,8 @@ def update_gui(self):
         payload = self.payloadImage()
         self.payload["image"] = json.dumps(payload)
         # Add position to payload
-        cat_x, cat_y, cat_z = self.hal.get_position()
```

Comment:
undo all changes in this file when drone_wrapper backend is ready

Reply:
sure